src/Carbon/Carbon.php

```diff
@@ -240,17 +242,19 @@ public function __construct($time = null, $tz = null)
     /**
      * Create a Carbon instance from a DateTime one.
      *
-     * @param \DateTime $dt
+     * @param \DateTime|\DateTimeImmutable|\DateTimeInterface $dt
      *
      * @return static
      */
-    public static function instance(DateTime $dt)
+    public static function instance($dt)
```

Comment:
And I agree with @lucasmichot that we should support only 5.5+ since anything older is not supported anymore by PHP at all.